src/py/mat3ra/made/tools/modify.py

```diff
@@ -462,3 +464,39 @@ def rotate_material(material: Material, axis: List[int], angle: float) -> Materi
     atoms.wrap()
 
     return Material(from_ase(atoms))
+
+
+def displace_interface(
```

Comment:
displace_interface_component(interface, displacement, component=InterfacePartsEnum.FILM, use_cartesian_coordinates=True)

Should use displacement in angstroms by default